01-shapefile-attributes.Rmd

```diff
@@ -101,13 +102,12 @@ point <- readOGR("boundaryFiles/HARV/", "HARVtower_UTM18N")
 
 ```
 
+##Querying shapefile metadata 
 Recall from the previous lesson that shapefile metadata can include a _class_, 
```

Comment:
I'm not sure what the _class_ (underscores are). we should probably use `` for code instead.